web/server/vue-cli/src/components/AnalysisInfoDialog.vue

```diff
@@ -272,6 +273,8 @@ export default {
 
       var analysisInfo = await this.loadAnalysisInfo(
         this.runId, this.runHistoryId, this.reportId);
+      decideNegativeCheckerStatusAvailability(
+        analysisInfo, this.runId, this.runHistoryId, this.reportId);
```

Comment:
This should be put below the `highlightedCmds` assignment such that it is visually together with the other API call that is related to the checker status.